cglib/src/test/java/net/sf/cglib/proxysample/ProxySample.java

```diff
@@ -64,7 +64,11 @@ public int getKala(float kalamees) {
         } catch (Throwable e) {
         	throw new UndeclaredThrowableException(e);
         }
-        return result.intValue();
+        if (result != null) {
```

Comment:
This seems defensive also, and it's also just a sample in a test.  If the underlying method the handler invocation returns isn't expected to return null, then we shouldn't check for null and we should let the NPE happen.